surveys/2024-annual-survey/translations/es.md

```diff
@@ -535,22 +535,22 @@ Type: select all that apply
 - Ciencia de datos
 - Implementación de base de datos
 - Aplicaciones de escritorio
-- Servicios para computadores de escritorio o teléfonos móviles
+- Computadores de escritorio o librerías de teléfonos móviles o servicios
 - Sistemas distribuídos
 - Dispositivos embebidos (con un sistema operativo)
-- Dispositivos embebidos (sin sistema operativo)
+- Dispositivos embebidos (metal puro)
```

Comment:
this was left as "bare metal" in a option of a previous question. Which one is more commonly used?